2024-06-15 10:40:20 | INFO | model_worker | args: Namespace(host='localhost', port=31002, worker_address='http://localhost:31002', controller_address='http://localhost:31001', model_path='/home/c205/workspace/models/Apollo-7B/', revision='main', device='cuda', gpus=None, num_gpus=1, max_gpu_memory=None, dtype=None, load_8bit=False, cpu_offloading=False, gptq_ckpt=None, gptq_wbits=16, gptq_groupsize=-1, gptq_act_order=False, awq_ckpt=None, awq_wbits=16, awq_groupsize=-1, enable_exllama=False, exllama_max_seq_len=4096, exllama_gpu_split=None, exllama_cache_8bit=False, enable_xft=False, xft_max_seq_len=4096, xft_dtype=None, model_names=None, conv_template=None, embed_in_truncate=False, limit_worker_concurrency=5, stream_interval=2, no_register=False, seed=None, debug=False, ssl=False)
2024-06-15 10:40:20 | INFO | model_worker | Loading the model ['Apollo-7B'] on worker 673717ad ...
2024-06-15 10:40:21 | ERROR | stderr | Loading checkpoint shards:   0%|                                                     | 0/8 [00:00<?, ?it/s]
2024-06-15 10:40:21 | ERROR | stderr | Loading checkpoint shards:  12%|█████▋                                       | 1/8 [00:00<00:02,  3.22it/s]
2024-06-15 10:40:22 | ERROR | stderr | Loading checkpoint shards:  25%|███████████▎                                 | 2/8 [00:00<00:01,  3.22it/s]
2024-06-15 10:40:22 | ERROR | stderr | Loading checkpoint shards:  38%|████████████████▉                            | 3/8 [00:00<00:01,  3.17it/s]
2024-06-15 10:40:22 | ERROR | stderr | Loading checkpoint shards:  50%|██████████████████████▌                      | 4/8 [00:01<00:01,  3.20it/s]
2024-06-15 10:40:22 | ERROR | stderr | Loading checkpoint shards:  62%|████████████████████████████▏                | 5/8 [00:01<00:00,  3.21it/s]
2024-06-15 10:40:23 | ERROR | stderr | Loading checkpoint shards:  75%|█████████████████████████████████▊           | 6/8 [00:01<00:00,  3.15it/s]
2024-06-15 10:40:23 | ERROR | stderr | Loading checkpoint shards:  88%|███████████████████████████████████████▍     | 7/8 [00:02<00:00,  3.17it/s]
2024-06-15 10:40:23 | ERROR | stderr | Loading checkpoint shards: 100%|█████████████████████████████████████████████| 8/8 [00:02<00:00,  3.48it/s]
2024-06-15 10:40:23 | ERROR | stderr | 
2024-06-15 10:40:25 | INFO | model_worker | Register to controller
2024-06-15 10:40:25 | ERROR | stderr | [32mINFO[0m:     Started server process [[36m147040[0m]
2024-06-15 10:40:25 | ERROR | stderr | [32mINFO[0m:     Waiting for application startup.
2024-06-15 10:40:25 | ERROR | stderr | [32mINFO[0m:     Application startup complete.
2024-06-15 10:40:25 | ERROR | stderr | [32mINFO[0m:     Uvicorn running on [1mhttp://localhost:31002[0m (Press CTRL+C to quit)
2024-06-15 10:40:55 | INFO | stdout | [32mINFO[0m:     127.0.0.1:56236 - "[1mPOST /worker_get_conv_template HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:40:55 | INFO | stdout | [32mINFO[0m:     127.0.0.1:56240 - "[1mPOST /model_details HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:40:55 | INFO | stdout | [32mINFO[0m:     127.0.0.1:56248 - "[1mPOST /count_token HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:40:55 | INFO | stdout | [32mINFO[0m:     127.0.0.1:56252 - "[1mPOST /worker_generate_stream HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:41:06 | INFO | stdout | [32mINFO[0m:     127.0.0.1:47654 - "[1mPOST /model_details HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:41:06 | INFO | stdout | [32mINFO[0m:     127.0.0.1:47664 - "[1mPOST /count_token HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:41:06 | INFO | stdout | [32mINFO[0m:     127.0.0.1:47674 - "[1mPOST /worker_generate_stream HTTP/1.1[0m" [32m200 OK[0m
2024-06-15 10:41:10 | INFO | model_worker | Send heart beat. Models: ['Apollo-7B']. Semaphore: Semaphore(value=4, locked=False). call_ct: 2. worker_id: 673717ad. 
2024-06-15 10:41:55 | INFO | model_worker | Send heart beat. Models: ['Apollo-7B']. Semaphore: Semaphore(value=5, locked=False). call_ct: 2. worker_id: 673717ad. 
2024-06-15 10:42:40 | INFO | model_worker | Send heart beat. Models: ['Apollo-7B']. Semaphore: Semaphore(value=5, locked=False). call_ct: 2. worker_id: 673717ad. 
2024-06-15 10:43:25 | INFO | model_worker | Send heart beat. Models: ['Apollo-7B']. Semaphore: Semaphore(value=5, locked=False). call_ct: 2. worker_id: 673717ad. 
2024-06-15 10:43:25 | ERROR | model_worker | heart beat error: HTTPConnectionPool(host='localhost', port=31001): Max retries exceeded with url: /receive_heart_beat (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7b9429e70290>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-06-15 10:43:30 | ERROR | model_worker | heart beat error: HTTPConnectionPool(host='localhost', port=31001): Max retries exceeded with url: /receive_heart_beat (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7b94298c3c50>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-06-15 10:43:33 | ERROR | stderr | [32mINFO[0m:     Shutting down
2024-06-15 10:43:33 | ERROR | stderr | [32mINFO[0m:     Waiting for application shutdown.
2024-06-15 10:43:33 | ERROR | stderr | [32mINFO[0m:     Application shutdown complete.
2024-06-15 10:43:33 | ERROR | stderr | [32mINFO[0m:     Finished server process [[36m147040[0m]
2024-06-15 10:43:33 | ERROR | stderr | Traceback (most recent call last):
2024-06-15 10:43:33 | ERROR | stderr |   File "/home/c205/anaconda3/envs/fastchat/lib/python3.11/asyncio/runners.py", line 118, in run
2024-06-15 10:43:33 | ERROR | stderr |     return self._loop.run_until_complete(task)
2024-06-15 10:43:33 | ERROR | stderr |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-06-15 10:43:33 | ERROR | stderr |   File "uvloop/loop.pyx", line 1517, in uvloop.loop.Loop.run_until_complete
2024-06-15 10:43:33 | ERROR | stderr | asyncio.exceptions.CancelledError
2024-06-15 10:43:33 | ERROR | stderr | 
2024-06-15 10:43:33 | ERROR | stderr | During handling of the above exception, another exception occurred:
2024-06-15 10:43:33 | ERROR | stderr | 
2024-06-15 10:43:33 | ERROR | stderr | Traceback (most recent call last):
2024-06-15 10:43:33 | ERROR | stderr |   File "<frozen runpy>", line 198, in _run_module_as_main
2024-06-15 10:43:33 | ERROR | stderr |   File "<frozen runpy>", line 88, in _run_code
2024-06-15 10:43:33 | ERROR | stderr |   File "/home/c205/workspace/FastChat-main/fastchat/serve/model_worker.py", line 425, in <module>
2024-06-15 10:43:33 | ERROR | stderr |     uvicorn.run(app, host=args.host, port=args.port, log_level="info")
2024-06-15 10:43:33 | ERROR | stderr |   File "/home/c205/anaconda3/envs/fastchat/lib/python3.11/site-packages/uvicorn/main.py", line 577, in run
2024-06-15 10:43:33 | ERROR | stderr |     server.run()
2024-06-15 10:43:33 | ERROR | stderr |   File "/home/c205/anaconda3/envs/fastchat/lib/python3.11/site-packages/uvicorn/server.py", line 65, in run
2024-06-15 10:43:33 | ERROR | stderr |     return asyncio.run(self.serve(sockets=sockets))
2024-06-15 10:43:33 | ERROR | stderr |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-06-15 10:43:33 | ERROR | stderr |   File "/home/c205/anaconda3/envs/fastchat/lib/python3.11/asyncio/runners.py", line 190, in run
2024-06-15 10:43:33 | ERROR | stderr |     return runner.run(main)
2024-06-15 10:43:33 | ERROR | stderr |            ^^^^^^^^^^^^^^^^
2024-06-15 10:43:33 | ERROR | stderr |   File "/home/c205/anaconda3/envs/fastchat/lib/python3.11/asyncio/runners.py", line 123, in run
2024-06-15 10:43:33 | ERROR | stderr |     raise KeyboardInterrupt()
2024-06-15 10:43:33 | ERROR | stderr | KeyboardInterrupt
